specifyweb/frontend/js_src/lib/components/Interactions/LoanReturn.tsx

```diff
@@ -105,7 +105,7 @@ function PreparationReturn({
   );
   const canDeselect = state.some(({ resolve }) => resolve > 0);
   const canSelectAll = state.some(
-    ({ resolve, unresolved }) => resolve < unresolved
+    ({ resolve, unresolved }) => resolve < unresolved || Number.isNaN(resolve)
```

Comment:
where is the NaN coming from?
it shouldn't be here - if it is, there may be a bug somewhere
and plus, even if you handle nan in this case, it won't be handled in all cases (i.e line 106 and others)

Reply:
when the user select the bulk value and delete it to enter manually a new number, for a fraction of time the value for resolve or return is NaN

Reply:
ah, yes, we have this issue in some places
ideally, the onValueChanged in Input.Number should be `number | undefined` rather than `number` (unless the input also has `isRequired={true}`, in which case `onValueChanged` should not be triggered while the input is empty